tutorials/advanced_features/Backends for Sequence Execution.ipynb

```diff
@@ -1,510 +1,610 @@
 {
```

Comment:
<p>I think this is worth highlighting, let's put this inside a "info" box</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/952/discussion/#comment-2444241332'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>